### DISCUSSION TITLE
Success with stromleser.de device

### DISCUSSION THREAD

**Discussion by wlma** [Charly-2/Tasmota-TD3511-MBus#3] [Show and tell]
Just some success-story:

Sucessfully use your script with a device from stromleser.de.

Only needed to flash a different tasmota32c3-image with your settings, and use pin 3 for RX and pin 1 for TX

Many thanks for your work!

**Comment by Charly-2** [comment]
Hi! 
Nice to hear that this is useful!
Are you using a TD3511 smart meter or other device?
